docs/modules/community/pages/support.adoc

```diff
@@ -0,0 +1,16 @@
+= Professional support
+
+Here is a list of companies which can provide you some support on James.
+
+== Linagora
+
+image::linagora.png[link=https://linagora.com]
```

Comment:
That's a side note for Linagora as an organization but we may want to have a detailed "James offering" page...

Reply:
I don't believe we have such a dedicated page yet though right?